browser/safe_browsing/sources.gni

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/components/ai_chat/core/common/buildflags/buildflags.gni")
+import("//brave/components/brave_wayback_machine/buildflags/buildflags.gni")
+import("//brave/components/playlist/common/buildflags/buildflags.gni")
+import("//brave/components/speedreader/common/buildflags/buildflags.gni")
+
+brave_browser_safe_browsing_sources = []
+
+brave_browser_safe_browsing_deps = [
```

Comment:
it doesn't look like a good way to fix such problems, i guess we would eventually end up with adding sources.gn like this one for all components. I guess it should be solved with public_deps in brave/browser/ui/BUILD.gn ?

Reply:
Fixed here https://github.com/brave/brave-core/pull/25340/commits/e3b74abbbd7fd114751f91ddcd00039fbfe6e082